include/l3.h

```diff
@@ -76,14 +88,27 @@ void l3__log_simple(const uint32_t loc, const char *msg,
                     const uint64_t arg1, const uint64_t arg2);
 #endif  // L3_LOC_ENABLED
 
+#ifdef __cplusplus
+}
+#endif
+
 /**
  * \brief Caller-macro to invoke L3 Fast logging.
  */
+#if __APPLE__
+
+#define l3_log_fast(msg, arg1, arg2) l3_log_simple(msg, arg1, arg2)
```

Comment:
FYI: For now -- Fast logging on Mac is falling-back to slow-logging interface.

Reply:
Yeah, I think this is fine.

Reply:
Same should go for 32-bit x86 and any other architectures.